composer.json

```diff
@@ -18,9 +18,11 @@
     "require": {
         "php": ">=5.5.9",
         "doctrine/common": "~2.4",
+        "fig/link-util": "^1.0",
```

Comment:
this dep is in require-dev in all components, but in require here, looks suspicious